addons/osfstorage/models.py

```diff
@@ -342,6 +352,14 @@ def is_checked_out(self):
                 pass
         return False
 
+    @property
+    def is_preprint_primary(self):
```

Comment:
The first time I wrote this I didn't like it. Having to put it in again still makes me sad. I'm sure there is a better Django-y way to do this (and the function that lives above it related to checked out files).